src/models/User.ts

```diff
@@ -91,4 +99,20 @@ export function createAssociations(): void {
     as: "session",
     foreignKey: { name: "email", field: "userEmail" }
   });
+  User.hasMany(Request.scope(RequestScope.ORGANIZATION_REGISTRATION), {
+    as: "organizationRegistrationRequests",
+    foreignKey: { name: "email", field: "userEmail" }
+  });
+  User.hasOne(Request.scope(RequestScope.ORGANIZATION_REGISTRATION), {
```

Comment:
it's ont clear which one between the user organization_requests is selected by this relationship

Reply:
(ping : )